core/src/main/java/org/apache/brooklyn/core/config/external/vault/VaultExternalConfigSupplier.java

```diff
@@ -84,18 +86,19 @@ public VaultExternalConfigSupplier(ManagementContext managementContext, String n
             this.version = -1; // satisfy the static analysis :)
             errors.add("'kv-api-version' must be either 1 or 2");
         }
-        recoverTryCount = NumberUtils.toInt(config.get("recoverTryCount"),10);
+        recoverTryCount = NumberUtils.toInt(config.get("recoverTryCount"), 10);
         mountPoint = config.get("mountPoint");
         if (Strings.isBlank(mountPoint) && this.version == 2) errors.add("missing configuration 'mountPoint'");
-        if (!Strings.isBlank(mountPoint) && this.version == 1) errors.add("'mountPoint' is only applicable when kv-api-version=2");
+        if (!Strings.isBlank(mountPoint) && this.version == 1)
+            errors.add("'mountPoint' is only applicable when kv-api-version=2");
```

Comment:
we're well in to the realm of personal preference here (and not worth changing) but most people in the project prefer braces

```
if (x) {
  errors.add(...);
}
```

for short things i and some people prefer one line

```
if (x) errors.add(...);
```

multi-line without braces (your change here) is not very common.

Reply:
thanks for the feedback, it was not actually me to make the change but intelliJ and its formatter, I personally actually find multiple line with even with no braces easier to read than a single line, but as mentioned this is personal preference really :) I will keep that in mind for future, thanks!